content/api/events-ingest.apib

```diff
@@ -81,6 +81,40 @@ Required Headers
             }
         }
 
+### Get Ingest Events [GET /v1/ingest/events/{id}]
+
+Downloads a gzip file of the ingest batch. 
```

Comment:
could also say "Downloads a gzip file of the ingested events" or
"Downloads ingested batch as exactly as it was uploaded" 
@jgzamora , what do you think of the language here?